cmd/integration/integration.go

```diff
@@ -317,8 +323,7 @@ containers:
 	ioutil.WriteFile(manifestFile.Name(), []byte(manifest), 0600)
 
 	// Wait for the mirror pod to be created.
-	hostname, _ := os.Hostname()
-	podName := fmt.Sprintf("static-pod-%s", hostname)
+	podName := "static-pod-localhost"
```

Comment:
FYI: I'm also touching this in #5785, but your PR will probably be merged sooner :)